progress/progress.go

```diff
@@ -99,6 +108,13 @@ func (b *Bar) Increment(text string) {
 		return
 	}
 
+	// Start the spinner if not already started
+	if b.ticker == nil {
+		fmt.Println()
+		b.ticker = time.NewTicker(100 * time.Millisecond)
+		go b.spin(b.ticker.C)
+	}
+
```

Comment:
I see this is moved such that now the spiner is always running, even in verbose mode (but as a noop due to the verbosity check being moved to the spinner's write).  I'm interested to know what bug this fixed, because fiddling with the line status is finnicky indeed, and of course it would be ideal to not start the goroutine if it's never going to be used.